neo4j-ogm-tests/neo4j-ogm-integration-tests/src/test/java/org/neo4j/ogm/drivers/bolt/driver/BoltDriverConfigurationTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2002-2019 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.ogm.drivers.bolt.driver;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.neo4j.driver.v1.Driver;
+import org.neo4j.driver.v1.GraphDatabase;
+import org.neo4j.ogm.config.Configuration;
+import org.neo4j.ogm.session.SessionFactory;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * @author Gerrit Meier
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(GraphDatabase.class)
+public class BoltDriverConfigurationTest {
+
+    @Before
+    public void setup() {
+        PowerMockito.mockStatic(GraphDatabase.class);
+        Driver driverMock = Mockito.mock(Driver.class);
+        Mockito.when(GraphDatabase.routingDriver(Mockito.any(), Mockito.any(), Mockito.any())).thenReturn(driverMock);
```

Comment:
```suggestion
        Mockito.when(GraphDatabase.routingDriver(Mockito.any(), Mockito.any(), Mockito.any()))
            .thenAnswer(Answers.RETURNS_MOCKS);
```

Plus the Answers import.